samples/sensor/bme280/src/main.c

```diff
@@ -13,7 +13,10 @@ void main(void)
 {
 	struct device *dev = device_get_binding("BME280");
 
-	printf("dev %p name %s\n", dev, dev->config->name);
```

Comment:
Would it not be better to leave this line in just move it to after the error trapping?